mjx/mujoco/mjx/_src/io.py

```diff
@@ -55,9 +55,9 @@ def _make_option(
     if o.solver not in set(types.SolverType):
       raise NotImplementedError(f'{mujoco.mjtSolver(o.solver)}')
 
-    for i in range(mujoco.mjtEnableBit.mjNENABLE):
-      if o.enableflags & 2**i:
-        raise NotImplementedError(f'{mujoco.mjtEnableBit(2 ** i)}')
+    # Check enable flags using enum pattern
+    if types.EnableBit(o.enableflags) not in set(types.EnableBit) and o.enableflags != 0:
+      raise NotImplementedError(f'{mujoco.mjtEnableBit(o.enableflags)}')
```

Comment:
I don't think `mujoco.mjtEnableBit(o.enableflags)` works when you have multiple bits set, you will get an error like this:

```
ValueError: Invalid int value for mjtEnableBit: 3
```